lib/features/login/presentation/controllers/login_biding.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:chorale_fva/features/login/presentation/controllers/login_controller.dart';
+import 'package:get/get.dart';
+
+class LoginBinding extends Bindings {
+  @override
+  void dependencies() {
+    // Get.lazyPut(() => SignUpUseCase(Get.find<AuthenticationRepositoryIml>()));
+    Get.put(
+      LoginController(),
+    );
+  }
+}
```

Comment:
Are you OK of the name 'login_biding' ? Is'n it login_binding ?

Reply:
Ah you're right ! I'll check it !